src/main/java/frc/robot/constants/ConstMotion.java

```diff
@@ -21,9 +26,28 @@ public class ConstMotion {
 
   public static final TalonFXConfiguration HOOD_CONFIGURATION = new TalonFXConfiguration();
 
+  public static final TalonFXConfiguration INTAKE_PIVOT_CONFIGURATION = new TalonFXConfiguration();
+
+  public static final Angle PANEL_ANGLE = Degrees.of(0);
+
+  public static final Angle RETRACT_INTAKE_ANGLE = Degrees.of(0);
+
+  public static final Angle DEPLOY_INTAKE_ANGLE = Degrees.of(0);
+
   static {
 
     HOOD_CONFIGURATION.MotorOutput.NeutralMode = NeutralModeValue.Brake;
     HOOD_CONFIGURATION.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+
+    INTAKE_PIVOT_CONFIGURATION.Slot0.kS = 0;
+    INTAKE_PIVOT_CONFIGURATION.Slot0.kG = 0;
+
+    INTAKE_PIVOT_CONFIGURATION.Slot0.kP = 0;
+
+    INTAKE_PIVOT_CONFIGURATION.Slot0.GravityType = GravityTypeValue.Arm_Cosine;
+
+    INTAKE_PIVOT_CONFIGURATION.Feedback.SensorToMechanismRatio = ((10 / 54) * (20 / 72) * (14 / 38) * (1 / 1)
+        * (1 / 1));
```

Comment:
Is 1/1 doing anything?
```suggestion
    INTAKE_PIVOT_CONFIGURATION.Feedback.SensorToMechanismRatio = ((10 / 54) * (20 / 72) * (14 / 38));
```